cmd/glusterblock/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	gfd "github.com/gluster/gluster-csi-driver/pkg/glusterblock"
```

Comment:
gfd (gluster file driver) to  fbd (gluster block driver)

  can we rename glusterblock as   glusterloopback

Reply:
can we call it 'glustervirtualblock' (gvb) instead? I am thinking of avoiding loop from code.